examples/videostream-cli/cli.py

```diff
@@ -133,6 +134,10 @@ def on_track(track):
     parser.add_argument("--verbose", "-v", action="count")
     parser.add_argument("--fps", type=int, help="fps you want to sample at")
     parser.add_argument("--save-dir", type=str, help="folder to save frames + latency data in")
+    parser.add_argument('--enable-prediction', action='store_true')
+    parser.add_argument("--output-fps", type=int, default=30, help="fps you want to save the video with")
```

Comment:
please shorten these two lines by making the help go on the next line.

Reply:
this and the next

Reply:
done